src/briefcase/commands/dev.py

```diff
@@ -207,7 +209,6 @@ def venv_name(self) -> str:
 
         :returns: Name for virtual environment directory
         """
-
         return "dev"
```

Comment:
It turns out the Python version numbering isn't really needed here - the venv path implies the Python version; and if you have 2 different Python environments with the same version, you need to re-install anyway. So - we might as well keep this simple and re-create on *any* environment change.